:memo: Add Release Schedule Information (#461)
moj-analytical-services/user-guidance: source/documentation/tools/ide-maintenance-policy/index.md

# Tooling Maintenance Policy

## Release Naming

Each release will adopt [Semantic Versioning](https://semver.org).

> In GitHub, we will include detailed release notes for each tool. You can find these here:

Reviewer: ```suggestion
We will publish release notes of all changes for each tool. You can find these on GitHub:
```

- [JupyterLab Releases](https://github.com/ministryofjustice/analytical-platform-jupyterlab/releases)
- [Visual Studio Code Releases](https://github.com/ministryofjustice/analytical-platform-visual-studio-code/releases)
- [RStudio Releases](https://github.com/ministryofjustice/analytical-platform-rstudio/releases)
- [Cloud Development Environment Base Releases](https://github.com/ministryofjustice/analytical-platform-jupyterlab/releases)

> 

Reviewer: Would suggest this so users don't mistake a summary for being exhaustive. What do you think?

In the Control Panel, there has historically been a loose naming convention for tooling releases. Going forward, we will adopt a clearer naming convention in the form:

`[release x.y.z] Tool Name x.y.z`

For example, for Visual Studio Code, this will appear as:

`[vscode 2.3.0] Visual Studio Code 2.3.0`

Semantic versions will match across both the GitHub repository and the Control Panel.

> Release notes in GitHub will include all changes, while release communications will provide a summary.
> 

Reviewer: ```suggestion
```
